avalon/blender/lib.py

```diff
@@ -0,0 +1,154 @@
+"""Standalone helper functions."""
+
+import contextlib
+from typing import Dict, List, Union
+
+import bpy
+
+from ..lib import logger
+from . import pipeline
+
+
+def imprint(node: bpy.types.bpy_struct_meta_idprop, data: Dict):
+    r"""Write `data` to `node` as userDefined attributes
```

Comment:
Not a problem, we've been using it in Pyblish since [the early days](https://github.com/pyblish/pyblish-base/blob/64a705faabbe81e20117d0997d39b3d285decb7b/pyblish/compat.py#L148). Sometimes, when you use special characters in a docstring, it's the only option, and there has been no consequence so far.